src/Valleysoft.DockerCredsProvider/CredsProvider.cs

```diff
@@ -5,25 +5,34 @@ namespace Valleysoft.DockerCredsProvider;
 public static class CredsProvider
 {
     public static Task<DockerCredentials> GetCredentialsAsync(string registry) =>
-        GetCredentialsAsync(registry, new FileSystem(), new ProcessService());
+        GetCredentialsAsync(registry, new FileSystem(), new ProcessService(), new Environment());
```

Comment:
To avoid needing to namespace-qualify the `System.Environment` all over the place, I think it'd be better to have a class name that didn't conflict since this particular class is referenced so few times. I suggest `EnvironmentWrapper`.

Reply:
Can do.